### PR TITLE
Scaffold vitest, migrate general.js as an example

### PR DESCRIPTION
**Pull request by pearofducks**

### CODE REVIEW

**Review by BalbinaK — APPROVED** [review]
Thanks for setting this migration up! Does it mean we can remove the commented bits in `general.js`?

**Comment by pearofducks** [comment]
Yes we can!

I'll probably wait to merge this until #4 lands. I don't want to cause any merge conflicts (most of this can be copy/pasta over if needed)!

**Comment by nwikstrand** [comment]
@pearofducks just merged #4 in order to not block this from going in. Lets gooooooo

**Pull request merged**
